OpenRA.Game/TraitTypeIndex.cs

```diff
@@ -0,0 +1,72 @@
+#region Copyright & License Information
+/*
+ * Copyright 2007-2017 The OpenRA Developers (see AUTHORS)
+ * This file is part of OpenRA, which is free software. It is made
+ * available to you under the terms of the GNU General Public License
+ * as published by the Free Software Foundation, either version 3 of
+ * the License, or (at your option) any later version. For more
+ * information, see COPYING.
+ */
+#endregion
+
```

Comment:
Okay, but Travis also reports all style issues: [Details](https://travis-ci.org/OpenRA/OpenRA/builds/246780380?utm_source=github_status&utm_medium=notification)

You'll have to update your branch with `TraitTypeIndex.cs` added to the .csproj to catch style errors of that file as well, though.